lightblue/src/main/java/org/esbtools/eventhandler/lightblue/LightblueNotificationRepository.java

```diff
@@ -270,25 +250,22 @@ private static NotificationEntity asEntity(Notification notification) {
                 "LightblueNotification is supported. Event type was: " + notification.getClass());
     }
 
-    static class ProcessingNotification implements Lockable {
+    static class ProcessingNotification {
         final String notificationId;
         final Date originalProcessingDate;
         final LightblueNotification notification;
 
-        static LockedResources<ProcessingNotification> parseLockableNotificationEntities(
+        static List<ProcessingNotification> parseNotificationEntities(
                 NotificationEntity[] entities,
                 LightblueRequester requester,
                 Map<String, ? extends NotificationFactory> notificationFactoriesByEntityName,
-                LockStrategy lockStrategy, Clock clock) {
-            List<LockedResource<ProcessingNotification>> acquiredLocks =
+                Clock clock) {
+           List<ProcessingNotification> processedNotifications =
                     new ArrayList<>(entities.length);
 
-            // Shuffling the entities means less lock contention among nodes which get similar
-            // batches.
-            List<NotificationEntity> shuffled = Arrays.asList(entities);
-            Collections.shuffle(shuffled);
+           List<NotificationEntity> notificationEntityList = Arrays.asList(entities);
```

Comment:
nitpick: Looks like these two lines (266, and 263) are missing one space from the indentation